selftests/unit/test_job.py

```diff
@@ -92,3 +92,28 @@ def post_tests(self):
         myjob.post_tests()
         self.assertEqual(myjob.unique_id[::-1],
                          open(os.path.join(myjob.logdir, "reversed_id")).read())
+
+    @unittest.skip("Issue described at https://trello.com/c/qgSTIK0Y")
```

Comment:
There is a good chance for this to be skipped forever. But if you monitor the card, we can revisit this when the card has some action. Something like:

```python
import json
from avocado.utils import download

...
...

def _is_issue_fixed():
    issue_url = 'https://trello.com/card/58079f5f3dea443d1800fe43/single-data-dir-get-tmp-dir-per-interpreter-breaks-multiple-jobs.json'
    issue = json.load(download.url_open(issue_url))
    if 'Done' in issue['actions'][0]['data']['list']['name']:
        return True
    return False

....
....

@unittest.skipIf(not _is_issue_fixed(), "Issue described at https://trello.com/c/qgSTIK0Y")
```